### ISSUE TITLE
Problems with Laravel 5

### ISSUE THREAD

**Issue by devonzara** [Zizaco/entrust#205]
I'm unable to use Entrust in Laravel 5.

When trying to use 1.3, I get:
```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Conclusion: remove laravel/framework 5.0.x-dev
    - zizaco/entrust 1.3.0 requires laravelbook/ardent ~2.4 -> satisfiable by laravelbook/ardent[v2.4.0, v2.4.1, v2.4.2].
    - zizaco/entrust 1.3.x-dev requires laravelbook/ardent ~2.4 -> satisfiable by laravelbook/ardent[v2.4.0, v2.4.1, v2.4.2].
    - laravelbook/ardent v2.4.0 requires illuminate/validation ~4.1 -> satisfiable by laravel/framework[4.1.x-dev, 4.2.x-dev], illuminate/validation[4.1.x-dev, 4.2.x-dev, v4.1.0, v4.1.1, v4.1.10, v4.1.11, v4.1.12, v4.1.13, v4.
1.14, v4.1.15, v4.1.16, v4.1.17, v4.1.18, v4.1.19, v4.1.2, v4.1.20, v4.1.21, v4.1.22, v4.1.23, v4.1.24, v4.1.25, v4.1.26, v4.1.27, v4.1.28, v4.1.29, v4.1.3, v4.1.30, v4.1.4, v4.1.5, v4.1.6, v4.1.7, v4.1.8, v4.1.9, v4.2.0-BETA1
, v4.2.1, v4.2.2, v4.2.3, v4.2.4, v4.2.5, v4.2.6, v4.2.7, v4.2.8, v4.2.9].
    - laravelbook/ardent v2.4.1 requires illuminate/validation ~4.1 -> satisfiable by laravel/framework[4.1.x-dev, 4.2.x-dev], illuminate/validation[4.1.x-dev, 4.2.x-dev, v4.1.0, v4.1.1, v4.1.10, v4.1.11, v4.1.12, v4.1.13, v4.
1.14, v4.1.15, v4.1.16, v4.1.17, v4.1.18, v4.1.19, v4.1.2, v4.1.20, v4.1.21, v4.1.22, v4.1.23, v4.1.24, v4.1.25, v4.1.26, v4.1.27, v4.1.28, v4.1.29, v4.1.3, v4.1.30, v4.1.4, v4.1.5, v4.1.6, v4.1.7, v4.1.8, v4.1.9, v4.2.0-BETA1
, v4.2.1, v4.2.2, v4.2.3, v4.2.4, v4.2.5, v4.2.6, v4.2.7, v4.2.8, v4.2.9].
    - laravelbook/ardent v2.4.2 requires illuminate/validation ~4.1 -> satisfiable by laravel/framework[4.1.x-dev, 4.2.x-dev], illuminate/validation[4.1.x-dev, 4.2.x-dev, v4.1.0, v4.1.1, v4.1.10, v4.1.11, v4.1.12, v4.1.13, v4.
1.14, v4.1.15, v4.1.16, v4.1.17, v4.1.18, v4.1.19, v4.1.2, v4.1.20, v4.1.21, v4.1.22, v4.1.23, v4.1.24, v4.1.25, v4.1.26, v4.1.27, v4.1.28, v4.1.29, v4.1.3, v4.1.30, v4.1.4, v4.1.5, v4.1.6, v4.1.7, v4.1.8, v4.1.9, v4.2.0-BETA1
, v4.2.1, v4.2.2, v4.2.3, v4.2.4, v4.2.5, v4.2.6, v4.2.7, v4.2.8, v4.2.9].
    - Can only install one of: laravel/framework[5.0.x-dev, 4.1.x-dev].
    - Can only install one of: laravel/framework[5.0.x-dev, 4.2.x-dev].
    - don't install illuminate/validation 4.1.x-dev|don't install laravel/framework 5.0.x-dev
    - don't install illuminate/validation 4.2.x-dev|don't install laravel/framework 5.0.x-dev
    - don't install illuminate/validation v4.1.0|don't install laravel/framework 5.0.x-dev
    - don't install illuminate/validation v4.1.1|don't install laravel/framework 5.0.x-dev
    - don't install illuminate/validation v4.1.10|don't install laravel/framework 5.0.x-dev
    - don't install illuminate/validation v4.1.11|don't install laravel/framework 5.0.x-dev
    - don't install illuminate/validation v4.1.12|don't install laravel/framework 5.0.x-dev
    - don't install illuminate/validation v4.1.13|don't install laravel/framework 5.0.x-dev
    - don't install illuminate/validation v4.1.14|don't install laravel/framework 5.0.x-dev
    - don't install illuminate/validation v4.1.15|don't install laravel/framework 5.0.x-dev
    - don't install illuminate/validation v4.1.16|don't install laravel/framework 5.0.x-dev
    - don't install illuminate/validation v4.1.17|don't install laravel/framework 5.0.x-dev
    - don't install illuminate/validation v4.1.18|don't install laravel/framework 5.0.x-dev
    - don't install illuminate/validation v4.1.19|don't install laravel/framework 5.0.x-dev
    - don't install illuminate/validation v4.1.2|don't install laravel/framework 5.0.x-dev
    - don't install illuminate/validation v4.1.20|don't install laravel/framework 5.0.x-dev
    - don't install illuminate/validation v4.1.21|don't install laravel/framework 5.0.x-dev
    - don't install illuminate/validation v4.1.22|don't install laravel/framework 5.0.x-dev
    - don't install illuminate/validation v4.1.23|don't install laravel/framework 5.0.x-dev
    - don't install illuminate/validation v4.1.24|don't install laravel/framework 5.0.x-dev
    - don't install illuminate/validation v4.1.25|don't install laravel/framework 5.0.x-dev
    - don't install illuminate/validation v4.1.26|don't install laravel/framework 5.0.x-dev
    - don't install illuminate/validation v4.1.27|don't install laravel/framework 5.0.x-dev
    - don't install illuminate/validation v4.1.28|don't install laravel/framework 5.0.x-dev
    - don't install illuminate/validation v4.1.29|don't install laravel/framework 5.0.x-dev
    - don't install illuminate/validation v4.1.3|don't install laravel/framework 5.0.x-dev
    - don't install illuminate/validation v4.1.30|don't install laravel/framework 5.0.x-dev
    - don't install illuminate/validation v4.1.4|don't install laravel/framework 5.0.x-dev
    - don't install illuminate/validation v4.1.5|don't install laravel/framework 5.0.x-dev
    - don't install illuminate/validation v4.1.6|don't install laravel/framework 5.0.x-dev
    - don't install illuminate/validation v4.1.7|don't install laravel/framework 5.0.x-dev
    - don't install illuminate/validation v4.1.8|don't install laravel/framework 5.0.x-dev
    - don't install illuminate/validation v4.1.9|don't install laravel/framework 5.0.x-dev
    - don't install illuminate/validation v4.2.0-BETA1|don't install laravel/framework 5.0.x-dev
    - don't install illuminate/validation v4.2.1|don't install laravel/framework 5.0.x-dev
    - don't install illuminate/validation v4.2.2|don't install laravel/framework 5.0.x-dev
    - don't install illuminate/validation v4.2.3|don't install laravel/framework 5.0.x-dev
    - don't install illuminate/validation v4.2.4|don't install laravel/framework 5.0.x-dev
    - don't install illuminate/validation v4.2.5|don't install laravel/framework 5.0.x-dev
    - don't install illuminate/validation v4.2.6|don't install laravel/framework 5.0.x-dev
    - don't install illuminate/validation v4.2.7|don't install laravel/framework 5.0.x-dev
    - don't install illuminate/validation v4.2.8|don't install laravel/framework 5.0.x-dev
    - don't install illuminate/validation v4.2.9|don't install laravel/framework 5.0.x-dev
    - Installation request for laravel/framework ~5.0 -> satisfiable by laravel/framework[5.0.x-dev].
    - Installation request for zizaco/entrust ~1.3@dev -> satisfiable by zizaco/entrust[1.3.0, 1.3.x-dev].
```

When trying to use 1.2.5, I get:
```
Executing command (CWD): php artisan clear-compiled
{
    "error":{
        "type":"Symfony\\Component\\Debug\\Exception\\FatalErrorException",
        "message":"Class 'Zizaco\\Entrust\\MigrationCommand' not found",
        "file":"{REMOVED}\\vendor\\zizaco\\entrust\\src\\Entrust\\EntrustServiceProvider.php",
        "line":58
    }
}
Script php artisan clear-compiled handling the post-update-cmd event returned with an error
```

**Comment by mihai-mocanu** [comment]
I am having trouble installing it on laravel ~5.0 as well. I got the same error outpus as @devonzara and when i tried the fix suggested #196 like this: 
```
"require": {
    "laravel/framework": "~5.0",
    "laravelbook/ardent": "dev-l5 as 2.4.2",
    "zizaco/entrust": "1.3.*@dev"
  },
```

I get 
```
Problem 1
    - The requested package laravelbook/ardent could not be found in any version, there may be a typo in the package name.
  Problem 2
    - zizaco/entrust 1.3.x-dev requires laravelbook/ardent ~2.4 -> no matching package found.
    - zizaco/entrust 1.3.0 requires laravelbook/ardent ~2.4 -> no matching package found.
    - Installation request for zizaco/entrust 1.3.*@dev -> satisfiable by zizaco/entrust[1.3.0, 1.3.x-dev].
```

I need this package on a project that I can't hold anymore. I know laravel 5 isn't out yet .. but i really have to start working so i must go with the dev versions.

Thanks!

**Comment by andrewelkins** [comment]
Great! Checkout the laravel-5 branch too
